app/aboot/aboot.c

```diff
@@ -5531,10 +5531,10 @@ void aboot_init(const struct app_descriptor *app)
 	}
 	if (!boot_into_fastboot)
 	{
-		if (keys_get_state(KEY_HOME) || keys_get_state(KEY_VOLUMEUP))
+		if (keys_get_state(KEY_VOLUMEUP))
 			boot_into_recovery = 1;
 		if (!boot_into_recovery &&
-			(keys_get_state(KEY_BACK) || keys_get_state(KEY_VOLUMEDOWN)))
+			(keys_get_state(KEY_HOME) || keys_get_state(KEY_BACK) || keys_get_state(KEY_VOLUMEDOWN)))
```

Comment:
In general, behavioral changes are supposed to be only made conditionally in the new LK branch. In this case it might be okay, since AFAICT `KEY_HOME` is only used for very ancient platforms...